.github/actions/create-release-candidate-v1/action.yml

```diff
@@ -16,14 +16,18 @@ inputs:
     required: true
   version-locked:
     description: 'Whether or not the version should be locked to axe-core'
-    default: 'false'
     required: false
+    default: 'false'
   docs-repo:
     description: 'The name of the repo where the release notes live'
+    required: false
     default: 'null'
+  docs-labels:
+    description: 'Labels for release notes issue. Comma-delimited list of labels (e.x. "release,PRIORITY: high")'
```

Comment:
Is it expected the default will be no labels moving forward? If not we should add a default value here for `"release,PRIORITY: high"`

Reply:
as discussed with @Zidious this is not needed